R/recall_count.R

```diff
@@ -1,12 +1,14 @@
 #' count_recall_by_make
 #'
-#' @param make list string
-#' @param manufacturer A bool, with this flag we are asking for the manufacturer not the make
-#' @param start_year An integer
-#' @param end_year An integer
+#' Queries the website with a url that it generates from it's parameters.
```

Comment:
there's a discrepancy in how we're saying what functions do - some say that it queries the website with a url, other say it calls the API, should be consistent - should also capitalize API (remember that these will automatically form the user docs, so any typos etc will end up as part of the compiled user document).